### ISSUE TITLE
Newlines in code blocks are rendered as \n

### ISSUE THREAD

**Issue by hmenke** [NixOS/nixos-search#161]
The example for the NixOS option `nixpkgs.overlays` reads
```nix
let # fix the syntax highlighting by making this a valid Nix expression
      example = literalExample
        ''
          [
            (self: super: {
              openssh = super.openssh.override {
                hpnSupport = true;
                kerberos = self.libkrb5;
              };
            })
          ]
        '';
```
however this is rendered as

![test](https://user-images.githubusercontent.com/1903556/91381669-0f63aa00-e87c-11ea-8fa6-d205baa1c88f.png)

Link to the search: https://search.nixos.org/options?show=nixpkgs.overlays&query=overlays&from=0&size=15&sort=relevance&channel=unstable#disabled

**Comment by hmaurer** [comment]
@garbas I am interested in picking this one up.
One question: where can I find the code which generates the ElasticSearch index? Specifically where `option_example` is encoded. It appears to be encoded as a string, so `JSON.parse(data.option_example)` would give the desired result, but I want to make sure that assumption always holds.

**Comment by garbas** [comment]
@hmaurer the code that imports this is at https://github.com/NixOS/nixos-search/blob/master/import-scripts/import_scripts/channel.py#L435

Also for frontend we don't use Javascript, but Elm. 

And regarding JSON.parse, it is exactly what we are doing at the import time (also for options_default field) but as you can see in #175 this is not what we should be doing. I did it because it was an easy way to get majority of the values to look ok, but clearly there are problems.

**Comment by garbas** [comment]
There is also another similar problem with showing up example but connected to html escaping #105

**Comment by ncfavier** [comment]
This was fixed in #239.
